PyStableMotifs/AttractorRepertoire.py

```diff
@@ -60,7 +60,7 @@ def __init__(self):
         self.relevant_nodes = None
 
     @classmethod
-    def from_primes(cls,primes,max_simulate_size=20,max_stable_motifs=10000):
+    def from_primes(cls,primes,max_simulate_size=20,max_stable_motifs=10000,MPBN_update=False):
```

Comment:
Because new functions and arguments have been added, we need to update the docs before merging. Do you know how to run `Build_Manual.bat` (on Windows) or `Build_Manual.sh` (on Mac/Linux)? You will need to install `pandoc` and `XeLaTeX`. We can talk more about how to set this via email.

Reply:
Oh, and one more thing I forgot to mention: we should add the MPBN reference to the documentation here. Here's the ref.:
L Paulevé, J Kolčák, T Chatain, S Haar, "Reconciling qualitative, abstract, and scalable modeling of biological networks." Nat. Com. vol. 11, no. 4256 (2020)

Reply:
Where should I add the MPBN reference? I tried adding it whenever there's a MPBN_update parameter for a function.